interview-questions.md

```diff
@@ -15,54 +15,65 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: 
+  
+  Git is the technology that we use to code and is stored on our local computers. Github is the remote place that we store as our code acting like a cloud where we save information.
 
   Researched answer:
 
-
+  Git, the Global Information Tracker, is a open-source software distributed version control system. It is locally sourced, keeping all of the coding history on your local computer, used as a tool to write code. GitHub is web-based used for hosting git repositories remotely. This allows for developers to collaborate easier and save back-up copies of there code.
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: Boolean values, which equate to true and false, can be  accessed by using the following operators: ==, ===, >, <, <=, >=, ||, &&, !. 
 
   Researched answer:
 
-
+  Loose equality operators: == decides whether the to value are the same but do not have to be the same data type. 
+  Strict equality operators: === do the same as the loose operator but both values must be the same data types in order to return a true output.
+  Rational operators: >,<,<=,>=, return a boolean value by looking at relationship between the two values such as 10 > 7 would output true.
+  Logical operators: &&(and), ||(or), !(not) takes two statements and combine them together changing or adding conditions that have to be met in order to be true, for example 7 > 5 && 6 < 11 would have an output of true becuase BOTH conditions are met.
+  
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: Index's are the place holders inside of arrays. For example, Inside of an array as such, var num = [7, 9, 10, 11], index 0 would be equal to 7, index 1 would be equal to 9 and so on. So, we can use those index positions to call on which number/s we would like to use in our code
 
-  Researched answer:
+  Researched answer: 
 
+  Index's can be thought of as like an address for every element in an array. The index's start at the number zero and move from left to right.
 
 
 4. What is iteration?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: 
+  
+  Iteration is used when using loops, in a assignment I had this week I used iteration when creating a for loop that would run through the numbers 1 through 20. As it ran through the process of numbers the output would display "odd" for every odd number and keep the evens as numbers.
 
+  Researched answer: 
+  
+  Iteration is the name of the actual process that takes place when running the for loop. It preforms the action that you want your loop to run for the amount of time you designate it to or until a condition that you set is met.
```

Comment:
This can be doven into a bit deeper. Iteration is one of the most important concepts we use as dev to accomplish huge amounts of work.